MovieGallery/Pages/Search.cshtml

```diff
@@ -1,10 +1,10 @@
 ﻿@page
 @model MovieGallery.Pages.SearchModel
 @{
-    ViewData["Title"] = "Search by year";
+    ViewData["Title"] = "Search";
```

Comment:
Removed by year since the websites usually have only 'Search' and not making it specific to search by year.

Reply:
Could other searches be added to this page in the future?  I'd leave it as "Search", to expand that flexibility.